L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -277,6 +277,13 @@ namespace trklet {
     void setStripLength_PS(double stripLength_PS) { stripLength_PS_ = stripLength_PS; }
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
+    double overlapsize() const { return overlapsize_; }
+    long unsigned int numtracksperbin() const { return numtracksperbin_; }
+    unsigned int numtrackscomparedperbin() const { return numtrackscomparedperbin_; }
+
+    //Grabs the bin edges you need for duplicate removal bins
+    const std::vector<double>& varrinvbins() const { return varrinvbins_; }
+
```

Comment:
For all of these, please use camelcase, e.g. overlapSize() and overlapSize_. For some more pointers on standard CMS style, you can look here:
http://cms-sw.github.io/cms_coding_rules.html

Reply:
Fixed!

Reply:
I don't see the push?

Reply:
Sorry, was waiting to see if there was anything I needed to fix with making the curTracklet const before I made another push. I've pushed it now.